### DISCUSSION TITLE
Lesson 4: print(compiled_sol) syntaxerror

### DISCUSSION THREAD

**Discussion by 0xhoudini** [smartcontractkit/full-blockchain-solidity-course-py#1336] [Q&A] [answered]
![L4](https://user-images.githubusercontent.com/101376558/160215423-02ccbfaf-8fdf-4987-bd57-3b23f27424af.jpeg)
<img width="1435" alt="L4 2" src="https://user-images.githubusercontent.com/101376558/160215426-c3cac48f-59cb-4145-9801-01b53c2d43ed.png">

pls help guide me in the direction, i'm a bit lost.

**Comment by cromewar** [comment, marked as answer]
The solcx is a normal missing import, so you don't have to worry about.
Anyways did you choose the correct python interpreter on vscode?

**Reply by nameless-xbt** [reply]
i'm almost positive i downloaded the correct interpreter, but i'll double check. btw, wut do you mean by normal missing port??

**Reply by nameless-xbt** [reply]
^0xhoudini w/ new account

**Reply by cromewar** [reply]
Vscode have troubles to identify some libraries which are installed, for example If you install brownie with pipx instead of pip, vscode will complain but I'll still work.

**Reply by nameless-xbt** [reply]
makes sense, thx for the help

**Reply by cromewar** [reply]
Regarding to the deploy failing it said that is bad syntax on the print statement which I suppose you already fixed, Cheers.
